packages/util-dynamodb/package.json

```diff
@@ -24,14 +24,16 @@
     "tslib": "^2.5.0"
   },
   "devDependencies": {
-    "@aws-sdk/client-dynamodb": "*",
     "@tsconfig/recommended": "1.0.1",
     "concurrently": "7.0.0",
     "downlevel-dts": "0.10.1",
     "rimraf": "3.0.2",
     "typedoc": "0.23.23",
     "typescript": "~4.9.5"
   },
+  "peerDependencies": {
+    "@aws-sdk/client-dynamodb": "^3.0.0"
```

Comment:
moved to peer instead of dependency because this is how lib-dynamodb does it.

we can decide to move lib-dynamodb's client-dynamodb to a hard dependency in a later PR